src/modules/connections/utils/index.ts

```diff
@@ -0,0 +1,15 @@
+export const DataBaseSelectTablesAndColumnsQuery = {
+  postgres: `
+    SELECT tablename AS table, column_name AS column
+    FROM pg_catalog.pg_tables
+    INNER JOIN information_schema.columns
+    ON table_name = tablename
+    WHERE table_schema = 'public'
+  `
+}
+
+export const formatTablesAndColumnsResponse = (data) => data.reduce((acc, { column, table}) => {
```

Comment:
Please, add parentheses to make it more readable
Same for if else 

And can you add types here?